public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -65,32 +65,41 @@ <h5 class="col-xs-12 box-title">
 
       <div class="results-grid" *ngIf="items.length > 0" >
-        <div class="row" *ngIf="items.length === 1 && items[0].standardConcept != 'S'">
+        <div class="row" *ngIf="items.length === 1 && items[0].standardConcept != 'S' && standardConcepts.length > 0">
           <div class="col-xs-12 db-alert">
             <clr-alert  [clrAlertClosable]="false" [clrAlertType]="info">
               Note: {{items[0].vocabularyId}} {{items[0].conceptCode}} "{{items[0].conceptName}}"
-              maps to a Standard Vocabulary {{standardConcepts[0].vocabularyId}} {{standardConcepts[0].conceptCode}}
-              and any listed below. Standard vocabularies capture data across a variety of source vocabularies.
+              maps to Standard Vocabulary {{standardConcepts[0].vocabularyId}} {{standardConcepts[0].conceptCode}}.
+              Standard vocabularies capture data across a variety of source vocabularies.
             </clr-alert>
           </div>
         </div>
         <div class="row">
-          <div class="col-md grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
-          <div class="col-md grid-header flex-xs-middle"> Synonyms </div>
-          <div class="col-md-2 grid-header flex-xs-middle"> Participants of <b> {{totalParticipants | number}}</b> </div>
+          <div class="col-md-3 grid-header flex-xs-middle"> {{ehrDomain.name}}</div>
+            <div class="col-md-4 grid-header flex-xs-middle"> Synonyms </div>
+          <div class="col-md-2 grid-header flex-items-xs-right"> Participants of <b> {{totalParticipants | number}}</b> </div>
           <div class="col-md-2 grid-header flex-xs-middle info-text">
             Vocabulary Code
             <clr-icon class="" shape="info-standard" class="is-solid"></clr-icon>
           </div>
           <div class="col-md-1 grid-header flex-xs-middle info-text"> View graphs </div>
-
         </div>
         <div class="row expandable-grid-row"
-             *ngFor="let r of items.concat(standardConcepts), let i = index"
+             *ngFor="let r of standardConcepts.concat(items), let i = index"
              [ngClass]="{'expanded-grid-row': r.expanded}" >
-          <div class="col-md flex-xs-middle" innerHtml="{{i+1}}. {{r.conceptName | highlightSearch : prevSearchText}}
+          <div class="col-md-3 flex-xs-middle" *ngIf="searchResult.matchType === 'CONCEPT_CODE' && r.standardConcept !='S';else code_match_standard"
+               innerHtml="<b>Source</b> {{r.conceptName | highlightSearch : prevSearchText}}
+            -- {{r.conceptId}}"> </div>
+          <ng-template #code_match_standard>
+            <div class="col-md-3 flex-xs-middle" *ngIf="searchResult.matchType === 'CONCEPT_CODE' && r.standardConcept ==='S';else code_match_not_standard" innerHtml="<b>Standard</b> {{r.conceptName | highlightSearch : prevSearchText}}
+            -- {{r.conceptId}}"> </div>
+          </ng-template>
+          <ng-template #code_match_not_standard>
```

Comment:
Is this used?

Reply:
yeah, this is used for displaying the number and concept name if the match type is not concept code.